Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -797,15 +802,27 @@ extension SwiftLanguageService {
       additionalParameters: additionalCursorInfoParameters
     )
 
-    return cursorInfoResponse.refactorActions.compactMap {
+    var canInlineMacro = false
+
+    var refactorActions = cursorInfoResponse.refactorActions.compactMap {
       do {
         let lspCommand = try $0.asCommand()
+        canInlineMacro = $0.title == "Inline Macro"
         return CodeAction(title: $0.title, kind: .refactor, command: lspCommand)
       } catch {
         logger.log("Failed to convert SwiftCommand to Command type: \(error.forLogging)")
         return nil
       }
     }
+
+    if (canInlineMacro) {
+      let expandMacroCommand = try! ExpandMacroCommand(positionRange: params.range, textDocument: params.textDocument)
```

Comment:
We shouldn’t use `try!` because we don’t want to crash the entire SourceKit-LSP server just because some request failed. Checking really quickly, I think we can just make `asCommand` not throw. If that’s not possible, we should use `orLog` to log the error and then just not add the code action.